src/Categories/Bicategory/Construction/Bimodules/Tensorproduct/Associator/Naturality.agda

```diff
@@ -0,0 +1,332 @@
+    ∘ᵥ Bimodhom.α f₃ ◁ (F₂' ∘₁ F₁'))
+    ∘ᵥ F₃ ▷ Bimodhom.α f₂ ⊚₁ Bimodhom.α f₁)
+    ∘ᵥ associator.from ≈⟨ refl⟩∘⟨ (◁-▷-exchg ⟩∘⟨refl) ⟩∘⟨refl ⟩
+
+    Coequalizer.arr F₃'⊗[F₂'⊗F₁']
+    ∘ᵥ ((Bimodhom.α f₃ ◁ Coequalizer.obj F₂'⊗F₁'
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂'⊗F₁')
+    ∘ᵥ F₃ ▷ Bimodhom.α f₂ ⊚₁ Bimodhom.α f₁)
+    ∘ᵥ associator.from ≈⟨ refl⟩∘⟨ assoc₂ ⟩∘⟨refl ⟩
+
+    Coequalizer.arr F₃'⊗[F₂'⊗F₁']
+    ∘ᵥ (Bimodhom.α f₃ ◁ Coequalizer.obj F₂'⊗F₁'
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂'⊗F₁'
+    ∘ᵥ F₃ ▷ Bimodhom.α f₂ ⊚₁ Bimodhom.α f₁)
+    ∘ᵥ associator.from ≈⟨ refl⟩∘⟨ (refl⟩∘⟨
+                          ▷-resp-sq αSqf₂⊗f₁) ⟩∘⟨refl ⟩
+
+    Coequalizer.arr F₃'⊗[F₂'⊗F₁']
+    ∘ᵥ (Bimodhom.α f₃ ◁ Coequalizer.obj F₂'⊗F₁'
+    ∘ᵥ F₃ ▷ Bimodhom.α (f₂ ⊗₁ f₁)
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂⊗F₁)
+    ∘ᵥ associator.from ≈⟨ refl⟩∘⟨ sym-assoc₂ ⟩∘⟨refl ⟩
+
+    Coequalizer.arr F₃'⊗[F₂'⊗F₁']
+    ∘ᵥ ((Bimodhom.α f₃ ◁ Coequalizer.obj F₂'⊗F₁'
+    ∘ᵥ F₃ ▷ Bimodhom.α (f₂ ⊗₁ f₁))
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂⊗F₁)
+    ∘ᵥ associator.from ≈⟨ refl⟩∘⟨ (⟺ ∘ᵥ-distr-⊚ ⟩∘⟨refl) ⟩∘⟨refl ⟩
+
+    Coequalizer.arr F₃'⊗[F₂'⊗F₁']
+    ∘ᵥ ((Bimodhom.α f₃ ∘ᵥ id₂)
+        ⊚₁ (id₂ ∘ᵥ Bimodhom.α (f₂ ⊗₁ f₁))
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂⊗F₁)
+    ∘ᵥ associator.from ≈⟨ refl⟩∘⟨ (identity₂ʳ ⟩⊚⟨ identity₂ˡ ⟩∘⟨refl) ⟩∘⟨refl ⟩
+
+    Coequalizer.arr F₃'⊗[F₂'⊗F₁']
+    ∘ᵥ (Bimodhom.α f₃ ⊚₁ Bimodhom.α (f₂ ⊗₁ f₁)
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂⊗F₁)
+    ∘ᵥ associator.from ≈⟨ (refl⟩∘⟨ assoc₂) ⟩
+
+    Coequalizer.arr F₃'⊗[F₂'⊗F₁']
+    ∘ᵥ Bimodhom.α f₃ ⊚₁ Bimodhom.α (f₂ ⊗₁ f₁)
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂⊗F₁
+    ∘ᵥ associator.from ≈⟨ sym-assoc₂ ⟩
+
+    (Coequalizer.arr F₃'⊗[F₂'⊗F₁']
+    ∘ᵥ Bimodhom.α f₃ ⊚₁ Bimodhom.α (f₂ ⊗₁ f₁))
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂⊗F₁
+    ∘ᵥ associator.from ≈⟨ αSqf₃⊗[f₂⊗f₁] ⟩∘⟨refl ⟩
+
+    (Bimodhom.α (f₃ ⊗₁ (f₂ ⊗₁ f₁))
+    ∘ᵥ Coequalizer.arr F₃⊗[F₂⊗F₁])
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂⊗F₁
+    ∘ᵥ associator.from ≈⟨ assoc₂ ⟩
+
+    Bimodhom.α (f₃ ⊗₁ (f₂ ⊗₁ f₁))
+    ∘ᵥ Coequalizer.arr F₃⊗[F₂⊗F₁]
+    ∘ᵥ F₃ ▷ Coequalizer.arr F₂⊗F₁
+    ∘ᵥ associator.from ≈⟨ refl⟩∘⟨ hexagon {B₃} {B₂} {B₁} ⟩
+
+    Bimodhom.α (f₃ ⊗₁ (f₂ ⊗₁ f₁))
+    ∘ᵥ α⇒⊗ {B₃} {B₂} {B₁}
+    ∘ᵥ Coequalizer.arr [F₃⊗F₂]⊗F₁
+    ∘ᵥ Coequalizer.arr F₃⊗F₂ ◁ F₁ ≈⟨ sym-assoc₂ ⟩
+
+    (Bimodhom.α (f₃ ⊗₁ (f₂ ⊗₁ f₁))
+    ∘ᵥ α⇒⊗ {B₃} {B₂} {B₁})
+    ∘ᵥ Coequalizer.arr [F₃⊗F₂]⊗F₁
+    ∘ᵥ Coequalizer.arr F₃⊗F₂ ◁ F₁ ≈⟨ sym-assoc₂ ⟩
+
+    ((Bimodhom.α (f₃ ⊗₁ (f₂ ⊗₁ f₁))
+    ∘ᵥ α⇒⊗ {B₃} {B₂} {B₁})
+    ∘ᵥ Coequalizer.arr [F₃⊗F₂]⊗F₁)
+    ∘ᵥ Coequalizer.arr F₃⊗F₂ ◁ F₁ ∎
+
+    where
+      open hom.HomReasoning
+      open TensorproductOfHomomorphisms f₂ f₁ renaming (αSq to αSqf₂⊗f₁)
+      open TensorproductOfHomomorphisms f₃ f₂ renaming (αSq to αSqf₃⊗f₂)
+      open TensorproductOfHomomorphisms (f₃ ⊗₁ f₂) f₁ renaming (αSq to αSq[f₃⊗f₂]⊗f₁)
+      open TensorproductOfHomomorphisms f₃ (f₂ ⊗₁ f₁) renaming (αSq to αSqf₃⊗[f₂⊗f₁])
+
+  α⇒⊗-natural∘arr : (α⇒⊗ {B₃'} {B₂'} {B₁'}
```

Comment:
consider formatting like
```
foo :
  term1
  ≈
  term2
```
and similarly for equalities, i.e. don't indent by 20+ but instead by 2. And for this stuff, a width limit of 120 is probably fine (instead of just 80).